modules/nf-core/datavzrd/main.nf

```diff
@@ -0,0 +1,63 @@
+process DATAVZRD {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/datavzrd:2.36.12--bb93c8c988b7a9af':
+        'community.wave.seqera.io/library/datavzrd:2.36.12--593eb75e566b7f2a' }"
+
+    input:
+    tuple val(meta), file(config_file), file(table)
+
+    output:
+    tuple val(meta), path("${meta.id}"), emit: report
```

Comment:
@nvnieuwk I changed the output to the meta.id here as well, because changing to a simple wildcard `*` did not work. Any ideas?

Reply:
Can you change it to `${prefix}` instead since the `meta.id` is not always the prefix. You'll have to remove the `def` before the `prefix` to get this to work though :)